### PR TITLE
Blogging Module - Unwanted Exception Suppressing

### PR DESCRIPTION
**Pull request by cotur**
Example exceptions.

```
There is no such an entity. Entity type: Volo.Blogging.Blogs.Blog, id: favicon.ico
There is no such an entity. Entity type: Volo.Blogging.Blogs.Blog, id: robots.txt
There is no such an entity. Entity type: Volo.Blogging.Blogs.Blog, id: wp-login.php
```

### CODE REVIEW

File: modules/blogging/src/Volo.Blogging.Application/Volo/Blogging/Blogs/BlogAppService.cs [hunk 1]
```diff
@@ -32,6 +33,13 @@ public async Task<BlogDto> GetByShortNameAsync(string shortName)
 
             if (blog == null)
             {
+                FileInfo fi = new FileInfo(shortName);
+
+                if (!string.IsNullOrEmpty(fi.Extension))
+                {
+                    return null;
```
Comment (hikalkan) [inline modules/blogging/src/Volo.Blogging.Application/Volo/Blogging/Blogs/BlogAppService.cs:40]: Did you check if it is OK to return null from this method (check all usages)? We don't want "object reference not set..." exceptions.

**Comment by hikalkan** [comment]
What about the status of this PR? Is it still draft?

**Comment by cotur** [comment]
@hikalkan yes it is. I've just decided to what is best way to do for it.

File: modules/blogging/src/Volo.Blogging.Web/Pages/Blogs/Shared/Helpers/BlogNameControlHelper.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+{
+    public static class BlogNameControlHelper
+    {
+        public static readonly string[] ProhibitedFileExtensions = new string[] {".ico", ".txt", ".php"};
+
+        public static bool IsProhibitedFileFormatName(string blogShortName)
+        {
+            if (!string.IsNullOrWhiteSpace(blogShortName))
+            {
+                var fileInfo = new FileInfo(blogShortName);
```
Comment (hikalkan) [inline modules/blogging/src/Volo.Blogging.Web/Pages/Blogs/Shared/Helpers/BlogNameControlHelper.cs:14]: don't use FileInfo, but only use simple case insensitive comparison

**Pull request merged**